data/benefits/javascript/region-nouvelle-aquitaine-stage-etranger.yml

```
@@ -23,6 +23,6 @@ unit: €
 periodicite: mensuelle
 legend: maximum
 montant: 400
-link: https://les-aides.nouvelle-aquitaine.fr/amenagement-du-territoire/stages-letranger-public-post-bac
+link: https://les-aides.nouvelle-aquitaine.fr/jeunesse/stages-letranger-public-post-bac-annee-academique-2022-2023#:~:text=Crit%C3%A8res%20de%20s%C3%A9lection&text=Dur%C3%A9e%20de%202%20semaines%20cons%C3%A9cutives,sera%20faite%20pour%20les%20BTS).
 teleservice: https://cas.aquitaine.fr/cas/login?service=https%3A%2F%2Fmes-demarches.nouvelle-aquitaine.fr%2FcraPortailFO%2Flogin%2Fcas
-instructions: https://les-aides.nouvelle-aquitaine.fr/amenagement-du-territoire/stages-letranger-public-post-bac
+instructions: https://les-aides.nouvelle-aquitaine.fr/jeunesse/stages-letranger-public-post-bac-annee-academique-2022-2023#:~:text=Crit%C3%A8res%20de%20s%C3%A9lection&text=Dur%C3%A9e%20de%202%20semaines%20cons%C3%A9cutives,sera%20faite%20pour%20les%20BTS).
```

```suggestion
instructions: https://les-aides.nouvelle-aquitaine.fr/jeunesse/stages-letranger-public-post-bac-annee-academique-2022-2023
```

Pareil ici (je ne suis pas sûr du pourquoi ils ont mis ça mais ça change rien 🤷 )